tests/integration/test_keeper_zookeeper_converter/test.py

```diff
@@ -256,10 +258,31 @@ def test_simple_crud_requests(started_cluster, create_snapshots):
 
     copy_zookeeper_data(create_snapshots)
 
-    genuine_connection = get_genuine_zk()
-    fake_connection = get_fake_zk()
+    genuine_connection.stop()
+    genuine_connection.close()
 
-    compare_states(genuine_connection, fake_connection)
+    genuine_connection = get_genuine_zk(timeout=5)
+
+    fake_connection = get_fake_zk(timeout=5)
+    for conn in [genuine_connection, fake_connection]:
+        for i in range(200):
+            if not conn.exists("/test_ephemeral/0"):
+                break
+
+            time.sleep(0.1)
+        else:
+            raise Exception("ZK refuse to remove ephemeral nodes")
+
+    # After receiving close request zookeeper updates pzxid of ephemeral parent.
+    # Keeper doesn't receive such request (snapshot created before it) so it doesn't do it.
```

Comment:
I think Keeper doesn't update pzxid on remove at all.
Is there a reason for this?

Reply:
Yeah, but I think we can forget about it